### PR TITLE
Header

### PR DESCRIPTION
**Pull request by imdy3011**

### CODE REVIEW

**Review by jasce — CHANGES_REQUESTED** [review]
Screen Shot 2021-01-31 at 1 38 16 PM
Text and logo not vertically aligned

File: client/src/Components/HomePage/header.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* img {
+	width: 48px;
+	height: 48px;
+	margin: 20px;
+} */
+
+.parentDiv {
+	display: flex;
+	justify-content: space-between;
+	border-bottom: 1px solid black;
```
Comment (jasce) [inline client/src/Components/HomePage/header.css]: color of the border is not black in design

**Review by jasce — CHANGES_REQUESTED** [review]

**Review by sai5499 — APPROVED** [review]

**Review by sai5499 — COMMENTED** [review]
proportions or little incorrect(like positions of components)

File: client/src/Components/HomePage/header.css (hunk 1 above)
Comment (imdy3011) [reply client/src/Components/HomePage/header.css]: border part is done

**Review by jasce — APPROVED** [review]

**Pull request merged**